DiamondBlackjack/sv_blackjack.lua

```diff
@@ -40,6 +61,21 @@ AddEventHandler('playerDropped', function (reason)
     end
 end)
 
+RegisterCommand("debugtableserver",function()
+    print("blackjackTables")
+    print("===============")
+    print(dump(blackjackTables))
+    print("blackjackGameData")
+    print("===============")
+    print(dump(blackjackGameData))
+end)
+
+RegisterCommand("debugcarddata",function()
+    print("carddata")
+    print("===============")
+    print(dump(blackjackGameData[1024]))
+end)
+
```

Comment:
Why is this modified in this PR? Seems completely unrelated.

Reply:
can you help me I tried to add your script in esx server but it says this in console 

Error parsing script @DiamondBlackjack/sv_blackjack.lua in resource DiamondBlackjack: @DiamondBlackjack/sv_blackjack.lua:34: 'end' expected (to close 'function' at line 30) near 'if'
Failed to load script sv_blackjack.lua.